<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tela de registro</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>381</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLimparTudo">
      <property name="text">
       <string>Limpar tudo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCriarConta">
      <property name="text">
       <string>Criar nova conta</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtNome"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>CPF:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtCPF"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Senha:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtPassword"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
